### PR TITLE
ci: Add codeowner path to ci-docs.yaml file

### PR DESCRIPTION
**Pull request by vsiravar**
Issue #, if available:
Add codeowners file path to `ci-docs.yaml` to prevent CI from getting blocked. 
*Description of changes:*
Fixes: 
*Testing done:*

N/A

- [X] I've reviewed the guidance in CONTRIBUTING.md

#### License Acceptance

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by austinvazquez — APPROVED** [review]

File: .github/workflows/ci-docs.yaml [hunk 1]
```diff
@@ -15,12 +15,14 @@ on:
...
       - '**.md'
       - 'contrib/**'
+      - '.github/CODEOWNERS'
   pull_request:
     branches:
       - main
     paths:
       - '**.md'
       - 'contrib/**'
+      - 'github/CODEOWNERS'
```
Comment (austinvazquez) [inline .github/workflows/ci-docs.yaml:25]: Caught this on second glance. `.github`*

**Review by austinvazquez — CHANGES_REQUESTED** [review]

**Review by austinvazquez — APPROVED** [review]
LGTM thanks!

**Review by weikequ — APPROVED** [review]
LGTM! Thanks!

**Comment by vsiravar** [comment]
Flaky tests, no impact on this change since there is no code change or workflow change.

**Pull request merged**